### DISCUSSION TITLE
Student Verification

### DISCUSSION THREAD

**Discussion by muleyks** [community/community#140426] [GitHub Education]
### Hi everyone,

I want to take advantage of GitHub Pro for free by using my student status, but they asked for proof that I am a student. I uploaded a photo of my student ID, but they rejected it, saying it wasn't convincing enough to prove my student status. I resubmitted my student ID yesterday again, but I still haven't received a response. I want to withdraw my application and apply again with my university transcript, but I can't do that because I already have an ongoing application. I urgently need GitHub Pro because I will use it for my classes and my professor insists on GitHub co-pilot. What can I do about this?

**Comment by t1noo7** [comment]
Hi @muleyks  ,

If you wanna apply for the Student Developer Pack and your benefits are available, the first thing to do that's you should verify your information through your school email, and of course each university's email information is available in Github's database for them to compare. You need to enter correct the school email information. After receiving a valid verify email from the github system, your benefits is available to use (Copilot Student dev pack), as it comes with the pack. As for the duration, student benefits last for 2 years, after which you have to re-verify your student status to use continuously.

you can offer [here](https://education.github.com/pack/redeem/copilot-student).
In case you can't redeem it, let contact support by opening a ticket [here](https://support.github.com/contact/education), select `I am or would like to be a Partner for the ...` and explain them your issue.
